### PR TITLE
Fix copy to clipboard and deploy it everywhere

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

**Comment by gastaldi** [comment]
Looks Awesome! Here are my suggestions, which can be done in a separate PR if you agree: 

- Display the Clipboard icon only when hovering the snippet area
- Change the clipboard icon to a check ✔️ to indicate that the copy succeeded

**Comment by gsmet** [comment]
I did both.

The check icon is a cool idea. I wasn't happy about the result and I struggled with some tooltip libraries but in the end it wasn't really worth it to include a tooltip library just for that. But with the icon, it's perfect.

File: assets/javascript/copy.js [hunk 1]
```diff
@@ -19,3 +19,13 @@ codes.forEach((code) => {
...
 let clipboard = new ClipboardJS('.btn-copy');
+clipboard.on('success', function(e) {
+    e.clearSelection();
+    e.trigger.className = e.trigger.className.replace("fa-clipboard", "fa-check");
+    e.trigger.setAttribute("title", "Copied!")
+
+    setTimeout(function() {
+        e.trigger.className = e.trigger.className.replace("fa-check", "fa-clipboard");
+        e.trigger.setAttribute("title", "Copy to clipboard")
+    }, 1000);
```
Comment (gastaldi) [inline assets/javascript/copy.js:30]: I think this can wait a bit longer (maybe 3s)?

**Review by gastaldi — APPROVED** [review]
Excellent! Added a minor suggestion

**Comment by maxandersen** [comment]
This is awesome.  Works on mobile too. At least safari iOS :)

File: assets/javascript/copy.js (hunk 1 above)
Comment (gsmet) [reply assets/javascript/copy.js:30]: aaaah, GH showed your comment when I clicked the Merge button.

Comment (gsmet) [reply assets/javascript/copy.js:30]: I pushed an update with 2 seconds, I think it's a good compromise.

**Pull request merged**